luaui/Widgets/gui_comselect.lua

```diff
@@ -0,0 +1,128 @@
+function widget:GetInfo()
+	return {
+		name = "Commander Selector",
+		desc = "Allows for proper keybind selection of any flavour of commanders",
+		author = "Hornet, with some ZK raiding",
+		date = "03 June 2024",
+		license = "GNU GPL, v2 or later",
+		layer = 1001,
+		enabled = true,
+	}
+end
+
+
+
+local myTeamID = Spring.GetMyTeamID()
+local spGetUnitTeam = Spring.GetUnitTeam
+local spGetMyTeamID = Spring.GetMyTeamID
+local spSendCommands = Spring.SendCommands
+
+local specOld = false
+local spec = false
+local team
+
+
```

Comment:
Old remnants?
```suggestion
```

Reply:
Too many changes in succession does that, yes, removed.